src/typing_test.erl

```diff
@@ -92,6 +92,22 @@ should_run(Name, {exclude,Set}) -> not sets:is_element(Name, Set).
 
 simple_test_() ->
   WhatNot = [
+    % FIXME soundness
```

Comment:
what's the problem here?

Reply:
Internal tally bugs that need to be fixed before merging.

Reply:
Substitution was not applied correctly for the newly implemented lists.